clients/web/src/presentation/pages/dashboard/Dashboard.tsx

```diff
@@ -0,0 +1,17 @@
+/* eslint-disable no-console */
```

Comment:
?

Reply:
updated in 261abcebb25b1b380a34f4234d3d8ab047533c0d